crates/credentials/src/verifiable_credential.rs

```diff
@@ -130,6 +79,52 @@ pub struct VerifiableCredential {
     pub credential_subject: CredentialSubject,
 }
 
+impl From<VerifiableCredential> for JwtPayloadVerifiableCredential {
+    fn from(credential: VerifiableCredential) -> Self {
+        JwtPayloadVerifiableCredential {
+            context: credential.context,
+            id: Some(credential.id),
```

Comment:
I'm not sure if it matters, but should this be `None` if `id.is_empty()`? Same question for `issuer`, `issuance_date`, and `credential_subject`.